src/setup.py

```diff
@@ -28,10 +28,9 @@
     packages=find_packages(),
     include_package_data=False,
     classifiers=[
-        'License :: OSI Approved :: GNU General Puclic License (GPL)',
```

Comment:
This typo traces all the way back to cb280f0630a3f2d723cb09fef9828103b7d563bd

Reply:
Hah, nice!